main.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
We really should not have this file in our repo.

@ashildmy, can you add it to the `.gitignore`, remove it from the repo, and add the command to generate it in the `README`?